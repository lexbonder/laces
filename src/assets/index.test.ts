import * as dice from './index';

test('assets/index.ts should export dice images', () => {
    expect(dice).toMatchSnapshot();
});
