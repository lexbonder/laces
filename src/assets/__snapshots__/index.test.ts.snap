// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`assets/index.ts should export dice images 1`] = `
{
  "dice1": "/laces/src/assets/dice1.svg",
  "dice2": "/laces/src/assets/dice2.svg",
  "dice3": "/laces/src/assets/dice3.svg",
  "dice4": "/laces/src/assets/dice4.svg",
  "dice5": "/laces/src/assets/dice5.svg",
  "dice6": "/laces/src/assets/dice6.svg",
}
`;
